src/Microsoft.DocAsCode.Build.Engine/Incrementals/IncrementalBuildContext.cs

```diff
@@ -706,8 +718,8 @@ private bool GetCanVersionIncremental(IncrementalStatus buildInfoIncrementalStat
             }
             else
             {
-                IncrementalInfo.ReportStatus(false, IncrementalPhase.Build, message);
-                Logger.LogInfo($"Group: {Version}, build strategy: {buildStrategy}, details: {message}", code: buildStrategy);
+                IncrementalInfo.ReportStatus(false, IncrementalPhase.Build, message, fullBuildReasonCode);
+                Logger.LogInfo($"Group: {Version}, build strategy: {buildStrategy}, details: {message}, full build reason code: {fullBuildReasonCode}", code: buildStrategy);
```

Comment:
Also pass `fullBuildReasonCode` through `code` parameter.

As this will change the code from general `IsFullBuild` to concrete reason code, I'd like to invite @LucasDvp to review this change.

Reply:
build Strategy code is also needed, because the current *code logic* in Build is also parse the `code` phase.